### PR TITLE
feat(ui): Show conditional release comparisons

### PR DESCRIPTION
**Pull request by matejminar**
We now show charts based on the access to performance, discover, and session data existence.

- [x] format chart tooltip value
- [x] hide marklines on non-default time periods
- [x] show loading states while fetching event counts

### CODE REVIEW

File: static/app/views/releases/detail/overview/releaseComparisonChart/releaseSessionsChart.tsx [hunk 1]
```diff
@@ -159,31 +174,322 @@ class ReleaseSessionsChart extends React.Component<Props> {
     }
   }
 
+  getSeries(chartType: ReleaseComparisonChartType) {
+    const {releaseSessions, allSessions, release, location, project, theme} = this.props;
```
Comment (matejminar) [inline static/app/views/releases/detail/overview/releaseComparisonChart/releaseSessionsChart.tsx:178]: This is just moved from `releaseComparison/index`

File: static/app/views/releases/detail/overview/releaseComparisonChart/index.tsx [hunk 2]
```diff
@@ -349,207 +357,227 @@ function ReleaseComparisonChart({
...
-        ? displaySessionStatusPercent(diffCrashedUsers)
-        : null,
-      diffDirection: diffCrashedUsers ? (diffCrashedUsers > 0 ? 'up' : 'down') : null,
-      diffColor: diffCrashedUsers ? (diffCrashedUsers > 0 ? 'red300' : 'green300') : null,
-    },
-    {
+  if (hasHealthData) {
+    charts.push(
+      {
+        type: ReleaseComparisonChartType.CRASH_FREE_SESSIONS,
```
Comment (matejminar) [inline static/app/views/releases/detail/overview/releaseComparisonChart/index.tsx:363]: This is the same, just wrapped in conditionals `hasDiscover, hasPerformance, hasHealthData`.

File: static/app/views/releases/detail/overview/releaseComparisonChart/index.tsx [hunk 3]
```diff
@@ -1059,8 +809,10 @@ const Cell = styled('div')<{role?: ComparisonRow['role']}>`
     p.role &&
     ['parent', 'children'].includes(p.role) &&
     css`
-      border-bottom: 0 !important;
-      padding-bottom: 0;
+      && {
```
Comment (priscilawebdev) [inline static/app/views/releases/detail/overview/releaseComparisonChart/index.tsx:812]: 👍

**Review by priscilawebdev — APPROVED** [review]
lgtm

**Pull request merged**